packages/fxa-settings/src/components/PageAvatar/Add.tsx

```diff
@@ -2,34 +2,94 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import { useAlertBar } from '../../lib/hooks';
-import React from 'react';
-import { Localized } from '@fluent/react';
+import React, { useRef, useState } from 'react';
 import { RouteComponentProps, useNavigate } from '@reach/router';
-
+import { Localized, useLocalization } from '@fluent/react';
+import { useAlertBar } from '../../lib/hooks';
+import { onFileChange } from '../../lib/file-utils';
+import { CapturePath, HomePath } from '../../constants';
+import { useAccount } from '../../models';
 import AlertBar from '../AlertBar';
 import Avatar from '../Avatar';
-// import AvatarCropper from '../AvatarCropper';
+import AvatarCropper from '../AvatarCropper';
 import ButtonIcon from '../ButtonIcon';
 import FlowContainer from '../FlowContainer';
 import { ReactComponent as AddIcon } from './add.svg';
 import { ReactComponent as CameraIcon } from './camera.svg';
-import { CapturePath, HomePath } from '../../constants';
+import { ReactComponent as RemoveIcon } from './remove.svg';
+
+const buttonClass = `mx-2 text-grey-500
+hover:text-grey-600 hover:text-grey-600
+focus:text-grey-400`;
 
-function save() {
-  console.log('save avatar');
+function remove() {
+  console.log('remove avatar');
 }
 
 export const PageAddAvatar = (_: RouteComponentProps) => {
   const navigate = useNavigate();
-  // const account = useAccount();
+  const { l10n } = useLocalization();
+  const { avatarUrl } = useAccount();
+  const fileInputRef = useRef(null);
   const alertBar = useAlertBar();
-  // const [errorText, setErrorText] = useState<string>();
   // const [saveEnabled, setSaveEnabled] = useState<boolean>();
+  const [imageSrc, setImageSrc] = useState(undefined);
   const handleSubmit = async (ev: React.FormEvent<HTMLFormElement>) => {
     ev.preventDefault();
```

Comment:
in the upload success response handler please add a line like this https://github.com/mozilla/fxa/blob/main/packages/fxa-settings/src/components/PageDisplayName/index.tsx#L48 to notify the browser.

Reply:
I have this added in another branch for handling uploads, https://jira.mozilla.com/browse/FXA-1702